api/src/org/apache/cloudstack/usage/UsageTypes.java

```diff
@@ -42,6 +42,7 @@
     public static final int VM_DISK_BYTES_READ = 23;
     public static final int VM_DISK_BYTES_WRITE = 24;
     public static final int VM_SNAPSHOT = 25;
+    public static final int DYNAMIC_ROUTING_POLICY = 16;
```

Comment:
As a new type, please use 26 or above, in case type value = 16 is used by someone else (or externally)